src/workflows/events.py

```diff
@@ -69,7 +69,8 @@ def __init__(self, **params: Any):
         private_attrs = {}
         data = {}
         for k, v in params.items():
-            if k in self.model_fields:
+            # using __class__.model_fields because using model_fields directly is deprecated
```

Comment:
Nit: ironically this comment will keep the deprecated method mentioned forever 😅 it should be enough to look at the commit message for future reference